config/helm/crd_bootstrap_webhook.yaml

```diff
@@ -0,0 +1,22 @@
+# The following patch enables conversion webhook for CRD
```

Comment:
Patch for setting our conversion spec (overriding namespace part) in all bootstrap CRDs (label selector is used in kustomization.yaml to apply this to all bootstrap CRDs).